csr_check.go

```diff
@@ -289,10 +299,20 @@ func recentlyPendingCSRs(indexer cache.Indexer) int {
 			continue
 		}
 
-		if !isApproved(csr) {
-			pending++
+		if isDenied(csr) {
+			denied++
 		}
 	}
 
-	return pending
+	return denied
+}
+
+func isNodeBootstrapCSR(csr *certificatesv1beta1.CertificateSigningRequest) bool {
+	parsedCSR, err := parseCSR(csr)
+	if err != nil {
+		klog.Errorf("Failed to parse CSR: %v", err)
+		return false
+	}
+	nodeAsking, _:= validateCSRContents(csr, parsedCSR)
```

Comment:
Why do we ignore any potential error here?

Reply:
we should log warning i think. It will help in debugging if somehow approver is skipping the csrs which it should not be.